src/client.ts

```diff
@@ -633,6 +608,8 @@ export class CustomTypesClient {
 			}
 		}
 
-		throw new PrismicError("An invalid API response was returned", { url });
+		throw new InvalidAPIResponse("An invalid API response was returned", {
```

Comment:
🎉 #nice: The more specific error is a nice improvement. 🙂 And it's backward compatible!